### ISSUE TITLE
Update/create new snapshot designs for multiple committee types

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#4900]
**What we're after:**
We've made some calculation changes, so we need to update previous versions with updated language. We also need additional options for different committee types for comparison. 

**Completion criteria:**
- [ ] Share updated language for review and feedback
- [ ] Create additional mockups for new committees

**Comment by JonellaCulmer** [comment]
@PaulClark2 Draft updated language:

Hybrid PAC:
- Donut 1: Percent raised in **contributions** (glossary link)
- Donut 2: Percent raised in other federal **receipts** (glossary link) including receipts to a **non-contribution account** (glossary link)

SSF
- Donut 1: Percent raised from individuals and partnerships
- Donut 2: Percent raised from **contributions** (glossary link) from other federal **political committees** (glossary link) and affiliated SSF transfers

Super PAC
- Donut 1: Percent raised from individuals and partnerships
- Donut 2: Percent raised in **contributions** (glossary link) from parties and other federal **political committees** (glossary link)

**Comment by PaulClark2** [comment]
@JonellaCulmer for SSFs we've changed the raising comparison to: 

- Donut 1: Percent itemized individual contributions
- Donut 2: Percent unitemized individual contributions 

The language above reflects what's on the F3X and might not be the most user friendly.

**Comment by PaulClark2** [comment]
Language below raising donuts. 
Question for the Content Team: Would the glossary entry be for `individual contributions` and within that entry explain both itemized and unitemized or would we have two glossary entries. 

Hybrid PAC:
- Donut 1: Percent raised in **contributions** (glossary link)
- Donut 2: Percent raised in other federal **receipts** (glossary link) including receipts to a **non-contribution account** (glossary link)

SSF
- Donut 1: Percent raised from **itemized individual contributions** (glossary link)
- Donut 2: Percent raised from **unitemized individual contributions** (glossary link)

Super PAC
- Donut 1: Percent raised from individuals and partnerships
- Donut 2: Percent raised in **contributions** (glossary link) from parties and other federal **political committees** (glossary link)

**Comment by JonellaCulmer** [comment]
@PaulClark2 We don't currently have a glossary definition for `individual contributions` but that would be better than just `contributions`. We also don't have anything for `itemized` or `unitemized`. 

Another question for content team is the LOE for adding those to the glossary if necessary. cc: @dorothyyeager @kathycarothers @djgarr @bmathesonFEC

**Comment by dorothyyeager** [comment]
@kathycarothers @djgarr and I chatted about this - we aren't sure how you will get good definitions for "individual contributions" or "itemized/unitemized"

Individual contributions = Line 11(a)(i) = Contributions from human beings, but also partnerships, LLCs taxed as partnerships, and other unincorporated entities. That is pretty complicated for a caption and so that's why we had recommended just leaving it as contributions.

Itemized contributions should be anything from one source aggregating over $200 for the election year. However, many committees, including the biggest one of all, choose to itemize everything. So it's kind of evolved into a dictionary definition of "everything that a committee is required to itemize specifically on a report plus all the things they weren't but chose to do so anyway." I'm not sure how we get this past a legal review.

**Comment by PaulClark2** [comment]
I'm not sure if this helps, but it was part of an email response to a question about uniquely identifying contributors. 

> The FEC’s reporting forms each have a line (Forms 3 (Line 11(a)(i), 3P(Line 17(a)(i) and 3X(Line 11(a)(i))  for disclosing itemized contributions (other than loans) from individuals/persons (other than political committees). Individuals are humans, but persons aren’t necessarily humans. Our regulations define a person as an individual, partnership, political committee, corporation, labor organization or any other organization or group of persons, not including the federal government. 11 CFR 100.10. So, the individual contribution line includes humans, partnerships, Native American Tribes and any other organization or group of persons, not including the federal government. 

To me it seems worth the effort to try to define these terms. The "contribution" entry in the glossary doesn't really help anyone understand what an itemized or unitemized individual contribution is.. It explains the "contribution" part of the term but doesn't help at all with "itemized" or "individual."  think the "dictionary" definition of an itemized contribution is useful. 

My understanding is the itemization rules for earmarking committees are a little different. I'm happy to be corrected on this. 

@dorothyyeager for PACs and party committees isn't the itemization threshold aggregating over $200 for a calendar year and an election cycle?

**Comment by dorothyyeager** [comment]
You are right @PaulClark2 - it's a calendar year threshold for PACs/Parties. I meant to type that but it came out "election year".

You are right about earmarking too - we have a reporting example for PACs. https://www.fec.gov/help-candidates-and-committees/filing-pac-reports/earmarked-contributions/ In that text, itemization threshold is noted as "exceeding $200." (No calendar year or election cycle mentioned.) Looking at https://www.fec.gov/regulations/110-6/2021-annual-110#110-6-c-1-iv - that's right.

A thought about "individual": We do have a regulatory definition of person, as you noted, and it's in our glossary. Maybe instead of "individual", use "Contributions from persons" and link that glossary term? The term includes "individual" in it. It's not as plain language but it will work better for a donut caption than "Contributions from individuals, partnerships, LLCs, etc"



I get why you want to have a definition of "itemized contribution."

We don't have reporting examples for PACs or parties for contributions received - seems like we should add that to our list. cc @bmathesonFEC

**Comment by PaulClark2** [comment]
Thanks @dorothyyeager I'm up for a call next week. I think we can also consider defining "individual" and "itemized" separately? So, two glossary entries. We already have a glossary entry for "contribution." Anyway, we should try "itemized contribution" first and then maybe separate entries as a fallback.

**Comment by cnlucas** [comment]
Not moving forward with this feature at this time.
